assembly/pom.xml

```diff
@@ -20,7 +20,7 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.apache.spark</groupId>
-    <artifactId>spark-parent_2.11</artifactId>
+    <artifactId>spark-parent</artifactId>
```

Comment:
Empirically, it looks like the generated POMS which are used for publishing end up with all variables in the `<dependencies>` section having been expanded according to whichever profile was active at publication time. For example, in http://central.maven.org/maven2/org/apache/spark/spark-core_2.10/1.6.1/spark-core_2.10-1.6.1.pom, there are no references to `scala.binary.version` in the `<dependencies>` section. There _are_ references in the `<build>` section but I don't think that this is relevant to consumers of this POM since I don't expect people to depend on our child POMs in their own builds.